### DISCUSSION TITLE
Volume control

### DISCUSSION THREAD

**Discussion by mrimpulsive** [MonitorControl/MonitorControl#1505] [Q&A]
when I downloaded app it work on brightness but volume only shows like its working but volume doesn't change ? Any solutions so it actually changes in volume . Also computer completely crashed and I had to restart to get it usable . Anyone experience these situations. I am using a brand new MC mini and a LG 32UN550 monitor .

**Comment by waydabber** [comment]
Hi - MonitorControl does not support HDMI on the non-pro mini and might cause a crash. Recommend using [BetterDisplay](https://betterdisplay.pro) instead.
